manager/integration/tests/run.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
 if [[ $@ =~ "--junitxml=" ]] ; then 
-  flake8
+  flake8 >> /dev/null
```

Comment:
Why do we need to run flake8 at runtime? I would suggest having it at build time.

Reply:
Fixed, Thanks